neurom/features/morphology.py

```diff
@@ -164,8 +164,55 @@ def trunk_vectors(morph, neurite_type=NeuriteType.all):
             for n in iter_neurites(morph, filt=is_type(neurite_type))]
 
 
+def _str_to_plane(plane):
+    """Transform a plane string into a list of coordinates."""
+    if plane is not None:
+        coords = []
+        plane = plane.lower()
+        if "x" in plane:
+            coords.append(COLS.X)
+        if "y" in plane:
+            coords.append(COLS.Y)
+        if "z" in plane:
+            coords.append(COLS.Z)
+    else:  # pragma: no cover
+        coords = [COLS.X, COLS.Y, COLS.Z]
+    return coords
+
+
+def _sort_angle(p1, p2):
+    """Angle between p1-p2 to sort vectors."""
+    ang1 = np.arctan2(*p1[::-1])
+    ang2 = np.arctan2(*p2[::-1])
+    return ang1 - ang2
+
+
+def _spherical_from_vector(vect):
+    """Returns the spherical coordinates of a vector: phi, theta."""
+    x, y, z = vect
+
+    phi = np.arctan2(y, x)
+    theta = np.arccos(z / np.linalg.norm(vect))
+
+    return np.array([phi, theta])
+
+
+def _angles_to_pi_interval(angle):
```

Comment:
Functions: `_angles_to_pi_interval`, `_spherical_from_vector`, `_sort_angle` must be moved to `morphmath` or `geom/__init__.py` as public functions.

Reply:
ok